.github/workflows/release.yml

```
@@ -26,7 +27,7 @@ jobs:
       - name: Install dependencies
         run: |
           sudo apt-get update
-          sudo apt-get install -y autoconf automake build-essential cmake gettext gperf libtool-bin locales ninja-build pkg-config unzip
+          sudo apt-get install -y autoconf automake build-essential cmake gettext gperf libtool-bin locales ninja-build pkg-config unzip gcc-11
```

Nitpick: `gcc-11` at the end doesn't maintain the alphabetical ordering that already exists (same comment applies for the `appimage` job below).

```suggestion
          sudo apt-get install -y autoconf automake build-essential cmake gcc-11 gettext gperf libtool-bin locales ninja-build pkg-config unzip
```